tests/test_synced_collection.py

```diff
@@ -476,39 +499,42 @@ def test_reset(self, synced_list):
         with pytest.raises(ValueError):
             synced_list.reset(1)
 
+    @given(JSONDataStrategy)
     def test_insert(self, synced_list, testdata):
         synced_list.reset([1, 2])
         assert len(synced_list) == 2
         synced_list.insert(1, testdata)
         assert len(synced_list) == 3
         assert synced_list[1] == testdata
 
-    def test_reversed(self,  synced_list):
-        data = [1, 2, 3]
-        synced_list.reset([1, 2, 3])
-        assert len(synced_list) == 3
+    @given(st.lists(JSONDataStrategy, max_size=5))
```

Comment:
You use this strategy enough times that it should have a name. In general, I would create a name for any strategy that is used more than two times (with a comment line describing its usage and contents) at the top of the file.